content/en/continuous_integration/setup_tests/swift.md

```diff
@@ -236,6 +247,10 @@ For Network auto-instrumentation, you can configure these additional settings:
 
 You can also disable or enable specific auto-instrumentation in some of the tests from Swift or Objective-C by importing the module `DatadogSDKTesting` and using the class: `DDInstrumentationControl`.
 
+## Alternative Configuration 
```

Comment:
```suggestion
## Using Info.plist for configuration
```

Also, why is this before `Custom tags`? They cannot be defined like this?

Reply:
Changed the name and moved